src/Application/Cli/PreviewCommand.php

```diff
@@ -77,18 +78,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
+        if (!is_executable($input->getOption('livereload'))) {
+            $output->writeln('<error>Impossible to launch Livereload, did you forgot to install it with "npm install -g livereload"?</error>');
+
+            return 1;
+        }
```

Comment:
This is a `InputOption::VALUE_REQUIRED` with a default value so it's always set, right?